specification/hybridconnectivity/HybridConnectivity.Management/PublicCloud.tsp

```diff
@@ -376,10 +382,13 @@ interface PublicCloudConnectors {
   testPermissions is ArmResourceActionAsync<
     PublicCloudConnector,
     void,
-    Azure.ResourceManager.Foundations.OperationStatusResult
+    HybridConnectivityOperationStatus
```

Comment:
Looks like there is no impact to generated code / swagger spec, revert this change if yes.